user-name-disputes.md

```diff
@@ -0,0 +1,91 @@
+
+This document is a clarification of the acceptable behavior outlined in
+the [npm Code of Conduct](https://www.npmjs.com/policies/conduct), and
+nothing in this document should be interpreted to contradict any aspect
+of the npm Code of Conduct.
+
+## Steps to Take
+
+1. Get the email associated with the desired user name from the profile
+   page: `https://www.npmjs.com/~<user_name>`
+2. Email the author, CC <support@npmjs.com>
+3. After a few weeks, if there's no resolution, the npm support team
+  will work with you to reach one.
+
+## Description
+
+As our community grows, so does the number of claimed user names or scopes.
+Occasionally, a user tries to create a new personal user account or
+organization, only to discover that the desired scope has already been
+taken.
+
+1. Ricardo creates a personal user account, `coolestname`, planning to
+   become active within the JavaScript community. They start, but soon
+   move on to other platforms leaving their user account unused. Some time
+   later, Irina comes along and wants to use the `coolestname` name as it
+   matches what they use elsewhere.
+2. Ricardo creates a personal user account, `nextbiggestthing`. They're
+   not attached to the name, but thought it would be popular someday. 
+   Later, Irina attempts to create an organization with the
+   `nextbiggestthing` scope matching their company name, Next Biggest
+   Thing Co.
+
+Irina's claims to the already uses scopes is debatable, but their course
+of action is the same.
+
+1. When alerted that a scope has already been taken it could be one
+  of two things:
+  
+    A. **User Scope**: A profile page of an existing user account will
+      give Irina the email address of the owner of the user name 
+      `https://www.npmjs.com/~<user_name>`.
+    B **Organization Scope**: If no profile with the name you are trying
+      to use exists, it's likely that the name is associated with an
+      organization scope. You'll need to reach out to the support team
+      <support@npmjs.com> to identify the organization super-admin and
+      associated email.
+
+2. Write an email:
+    - explaining the terms of her claim to the name
+    - CC's npm support staff <support@npmjs.com>
+
+3. After a reasonable amount of time, if Ricardo has not responded, or if
+   Ricardo and Irina can't come to any sort of resolution, support
+   <support@npmjs.com> will follow up to help moderate and move the
+   conversation. ("Reasonable" is usually at least 4 weeks.)
+  
+## Power to the People
+
+In most cases so far, the parties involved have been able to
+reach an amicable resolution without any major intervention.  Most
+people really do want to be reasonable, and are probably not even
+aware that they're in your way.
+
+When humans solve their problems by talking to other humans with
+respect, everyone has the chance to end up feeling good about the
+interaction.
+
+## Exceptions
+
+Some things are not allowed.
+
+?????
```

Comment:
3 and 7 from the other policy.

If your name is a trademark, we will give it up if another party can prove they own the trademark. If multiple parties control conflict trademarks in different jurisdictions they can go to court and figure it out, but it's not our job.